frontend/src/components/RecipePreparation.js

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components/macro'
+
+export default function RecipePreparation({ingredients, preparationSteps}) {
+    return (
+        <PreparationGrid>
+            <div>
+                <ul>
+                    <p>Zutaten</p>
+                    {
+                        ingredients.map(ingredient => (
+                            <li key={ingredient.name}>{ingredient.amountAndUnit} {ingredient.name}</li>))
+                    }
+                </ul>
+            </div>
```

Comment:
I would extract a component

Reply:
I would like to keep this as one component for responsive styling and cohesive styling e.g. for headings